destination/destination.go

```diff
@@ -0,0 +1,95 @@
+// Copyright © 2023 Meroxa, Inc. & Yalantis
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package destination implements the destination logic of the Neo4j connector.
+package destination
+
+import (
+	"context"
+	"fmt"
+
+	sdk "github.com/conduitio/conduit-connector-sdk"
+	"github.com/neo4j/neo4j-go-driver/v5/neo4j"
+)
+
+// Writer is a writer interface needed for the [Destination].
+type Writer interface {
+	Write(ctx context.Context, record sdk.Record) error
+}
+
+// Destination Neo4j Connector persists records to a Neo4j.
+type Destination struct {
+	sdk.UnimplementedDestination
+
+	config Config
+	writer Writer
+	driver neo4j.DriverWithContext
+}
+
+// NewDestination creates a new instance of the [Destination].
+func NewDestination() sdk.Destination {
```

Comment:
maybe we can rename `NewDestination()` to `New()` . Because when export package it looks now `destination.NewDestination()`
What do you think about it?

Reply:
Yeah, this is a good point. Thank you!